packages/fetcher-atama/index.ts

```diff
@@ -33,45 +49,39 @@ export class FetcherAtama extends Fetcher<AtamaFetcherConfig> {
     includedPaths?: string[];
     excludedPaths?: string[];
   } = {}): Promise<string[]> {
-    const url = this.config.url
-      ? new URL(
-          `v1/${this.config.environment || 'prod'}/${
-            this.config.workspaceId
-          }/paths`,
-          this.config.url,
-        )
-      : new URL(
-          `https://cdn.atama.land/v1/${this.config.environment || 'prod'}/${
-            this.config.workspaceId
-          }/paths`,
-        );
+    const pathsInput: GetPathsInput = {
+      workspaceId: this.config.workspaceId,
+      environment: this.config.environment,
+    };
     if (includedPaths) {
-      url.searchParams.set('includedPaths', includedPaths?.join(','));
+      pathsInput.includedPaths = includedPaths?.join(',');
     }
 
     if (excludedPaths) {
-      url.searchParams.set('excludedPaths', excludedPaths?.join(','));
+      pathsInput.excludedPaths = excludedPaths?.join(',');
     }
 
     let result;
     try {
-      // eslint-disable-next-line no-console
-      console.debug(`Running request to get paths against ${url.toString()}`);
-
-      const response = await fetch(url.toString(), {
-        headers: {
-          Authorization: `Bearer ${this.config.apiKey}`,
+      const response = await this.runGraphQLRequest<{
+        getPaths: string[];
+      }>(
+        gql`
+          query GetPaths($pathsInput: GetPathsInput!) {
+            getPaths(pathsInput: $pathsInput)
+          }
+        `,
+        {
+          pathsInput,
         },
-      });
+      );
 
-      if (!response.ok) {
-        throw new AtamaFetcherError(response.status);
+      if (!response.getPaths) {
+        // TODO: figure out what the error was and return a different status?
+        throw new AtamaFetcherError(500);
```

Comment:
I think for now it's okay to just return a `500` even though it's not 100% correct. In the case of `getPaths` we either get paths or we don't. It's very boolean so I wouldn't worry about the actual status code.